english/karaoke/nope.md

```diff
@@ -0,0 +1,5 @@
+You walk into a bar and see karaoke is going on!
+
+[nope out](../marshmellow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.